templates/core/about/builds.html

```diff
@@ -29,6 +29,13 @@ <h4 id="setting-a-readme"> <a href="#setting-a-readme">Setting a README</a> </h4
     </p>
 
     <h4 id="detecting-docsrs"> <a href="#detecting-docsrs">Detecting Docs.rs</a> </h4>
+    <p>
+        To recognize Docs.rs from your Rust code, you can test for the <code>docsrs</code> cfg, e.g.:
```

Comment:
Should mention that this is only set in the terminal crate, not for dependencies.

Reply:
The documentation for `cargo rustdoc` [says](https://doc.rust-lang.org/nightly/cargo/commands/cargo-rustdoc.html#description):

> The specified target for the current package (or package specified by -p if provided) will be documented with the specified args being passed to the **final rustdoc invocation**. Dependencies will not be documented as part of this command.

----

So maybe we should also use "final rustdoc invocation"?

```diff
-        The `docsrs` cfg only applies to the "terminal" crate (aka the last crate build by Cargo).
-        It does not applies to dependencies or workspace members, only the "terminal" crate.
+        The `docsrs` cfg only applies to the final rustdoc invocation. It does not applies
+        to dependencies (even workspace one).
```

Reply:
`cargo rustc` also has this distinction and uses terms like in `cargo help rustc`:
- "current package" (vs "all of its dependencies")
- "final compiler invocation"

(`cargo help rustdoc` seems to gloss over this)

With a quick glance, I'm not seeing this distinction made elsewhere.

rust-lang/rfcs#3310 uses the term "root crate" for "selected packages" but our existing documentation refers to the "root crate" as the crate in the non-virtual workspace manifest.

Reply:
Thanks for the input. It seems like there isn't (yet) a name for this.

So for the purpose documenting the `docsrs` cfg I went ahead and updated the text to:

> The `docsrs` cfg only applies to the final rustdoc invocation (i.e. the crate currently being documented). It does not apply to dependencies (including workspace ones).